Added Instructions (#2)
LetsShop3DS/SaveMii: README.md

# SaveMii
NNID Archiver Service (source code)
## How do I setup this?

> This is a standard Django app,

Reviewer: ```suggestion
This is a standard Django app, you probably should know how to setup one.
```

> This is a standard Django app,

Reviewer: ```suggestion
This is a standard Django app. You probably should know how to setup one. But if you really need steps, here you go (for development setup only):
```
some changes + made everything in a single line, cleaner

also i'll write production steps later, because if someone followed what you wrote for a production setup, he would end up with a pretty damn terrible prod setup, since the db is SQLite3, and some settings would be missing.

Author: yeah would be dumb if its a dev build forever

> But if you really need steps, here you go :

Reviewer: ```suggestion

```

> ### Step one

Reviewer: ```suggestion

```
there is only one sooo

> - Clone the repository : `git clone https://github.com/LetsShop3DS/SaveMii.git`

Reviewer: ```suggestion
- First, clone the repository: `git clone https://github.com/LetsShop3DS/SaveMii.git`
```

> - Do `cd SaveMii`

Reviewer: ```suggestion
- After that, run `cd SaveMii`
```
idk better

> - Create the database ! `python3 manage.py makemigrations miiapp`

Reviewer: ```suggestion
- Now, let's create the database: `python3 manage.py makemigrations miiapp`
```
why- why should this be a happy phrase

Author: 🤷

> - Migrate it : `python3 manage.py migrate`

Reviewer: ```suggestion
- After that, we'll apply the migrations to the database: `python3 manage.py migrate`
```
clearer

> - Now for the CSS/JS : `python3 manage.py collectstatic` and type "yes"

Reviewer: ```suggestion

```
we dont need this for a dev setup

> - Now run the server with `python3 manage.py runserver IP-HERE:8000` for developpement and `python3 manage.py runserver IP-HERE:80` for production.

Reviewer: ```suggestion
- Now, run the server with `python3 manage.py runserver`.

**If everything went well, congrats, you now have a development setup! If it doesn't work, create an issue.**
```
add instructions if not working
